docs/reference/evaluation-api/using-the-evaluation-api.mdx

```diff
@@ -118,16 +122,84 @@ MyObject objectValue = client.getObjectValue<MyObject>("objectFlag", MyObjectIns
 
 ```csharp
 // get a boolean value
-var boolValue = client.GetBooleanValue("boolFlag", false);
+var boolValue = await client.GetBooleanValue("boolFlag", false);
 
 // get a string value
-var stringValue = client.GetStringValue("stringFlag", "default");
+var stringValue = await client.GetStringValue("stringFlag", "default");
 
 // get an integer value
-var intValue = client.GetIntegerValue("intFlag", 1);
+var intValue = await client.GetIntegerValue("intFlag", 1);
```

Comment:
This was misleading, so I'm awaiting it now to make it clear it's a task/promise.